install.sh

```diff
@@ -500,6 +516,21 @@ if should_install_command_line_tools; then
   ohai "The Xcode Command Line Tools will be installed."
 fi
 
+non_default_repos=""
+additional_shellenv_command=()
+if [[ "$HOMEBREW_BREW_DEFAULT_GIT_REMOTE" != "$HOMEBREW_BREW_GIT_REMOTE" ]]; then
+  ohai "HOMEBREW_BREW_GIT_REMOTE is set to a non-default URL:"
+  echo "${tty_underline}${HOMEBREW_BREW_GIT_REMOTE}${tty_reset} will be used for Homebrew/brew Git remote."
```

Comment:
Feels like a shell bell might be nice here too: 

Could split this into a function and use it:
https://github.com/Homebrew/install/blob/e2448b098d5519e605b04d4c81622b075f33117b/install.sh#L656-L659